test/controllers/observer_controller_test.rb

```diff
@@ -770,47 +770,14 @@ def test_show_observation_change_vote_anonymity
   def test_show_owner_id
     login(user_with_view_owner_id_true)
     obs = observations(:owner_only_favorite_ne_consensus)
-    get_with_dump(:show_observation, id: obs.id)
+    get(:show_observation, params: { id: obs.id })
```

Comment:
Why get rid of the `get_with_dump`?  We don't use it much, but it is useful to ensure compliance with HTML standards.

Reply:
Thanks again. I didn't realize why we did it. When I initially created the tests (years ago?) i was just imitating other tests. I took it out because I thought it would be slightly more efficient.
I'll put them back.

Reply:
Thanks!  Looks good.